app/js/app.js

```diff
@@ -90,21 +100,64 @@ nijelApp.config(['$stateProvider', '$httpProvider',
             })
             .state('admin', {
                 url: '/admin',
+                templateUrl: 'views/layout.html',
```

Comment:
For this particular state, we don't need a templateUrl and its associated view.

Reply:
This template forms the base template for the admin section hence the URL